sorc/ncep_post.fd/RQSTFLD.F

```diff
@@ -2656,6 +2656,19 @@ module RQSTFLD_mod
       DATA IFILV(978),AVBL(978),IQ(978),IS(978),AVBLGRB2(978)      &
      &                      /1,'HIM-8 AHI CH16 NNADR',118,109,     & !table 130
      &                       'HIM-8 AHI CH16 NON-NADIR BRTEMP'/
+      DATA IFILV(983),AVBL(983),IQ(983),IS(983),AVBLGRB2(983)      &
```

Comment:
@EdwardColon-NOAA May I know the reason that you assign new ID starting from 983 instead of 979?